lib/sqspoller/worker_task.rb

```diff
@@ -4,9 +4,6 @@
 module Sqspoller
   class WorkerTask
 
-    HEADERS = { 'Content-Type' => 'application/json',
```

Comment:
Why have you removed this ?

Reply:
it was moved upstream in a change that has not been committed yet...